script/Arcueid/Day 13/13_00_ARC13_3.txt

```diff
@@ -213,42 +226,49 @@ I won't be able to miss Roa's 'death' no matter what.
 [sha:ca7a6a6a90c03d679fc0cb96b02fd5ae1d0514b8]{
 -- Page 249, Offset 32356.
 -- ここから先は一人で行ってください。わたしは別行動をしますから」
-From here on, you're on your own. I have other business to take care of."
+// Reading 別行動 as attacking from another angle, rather than "I need to go run some errands"
```

Comment:
the other place this expression was used iirc was so shiki could yell at arc when he took her to the cinema.

maybe she's gonna go watch that french movie?